mavisp/mavisp.py

```diff
@@ -235,6 +236,13 @@ def main():
     dataset_tables_path = out_path / 'dataset_tables'
     dataset_tables_path.mkdir(exist_ok=True)
 
+    # Generate a csv file that contains the number of mutations and the date of the run
+    time = strftime("%Y-%m-%d", gmtime())
+    nb_mutations = len(mfs.dataset_table['mutations'].explode().unique())
```

Comment:
I think you should `pd.unique` them before `pd.explode`ing them; the same mutation can be present by chance in different proteins and still be unique. It might not be even necessary to `unique` them - please double check if we do so during parsing of the mutation list